torch/distributed/tensor/parallel/api.py

```diff
@@ -82,6 +87,14 @@ def parallelize_module(  # type: ignore[return]
 
     torch._C._log_api_usage_once("torch.distributed.tensor.parallel.parallelize_module")
 
+    # instantiate a TP RNG state tracker if it's not there
+    if (
+        is_rng_supported_mesh(device_mesh) and
+        not isinstance(random._rng_tracker, TensorParallelRNGTracker)
+    ):
+        random._rng_tracker = TensorParallelRNGTracker()
+        random._rng_tracker._manual_seed(device_mesh, base_seed=(1234 + 2718), tp_dim=tp_mesh_dim)
```

Comment:
why 1234 + 2718?

Reply:
`1234` is the default seed used and `2718` is the "magic number" added to tensor parallel seed in Megatron. I will move `2718` back to TP RNG tracker and leave `1234` here.